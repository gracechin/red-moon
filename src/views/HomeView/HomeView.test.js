import { render, screen } from "@testing-library/react";
import { HomeView } from "./index";

describe("<HomeView />", () => {
  it("should render view", () => {
    render(<HomeView />);
    expect(screen.getByTestId("home-view")).toMatchSnapshot();
  });

  it("should render navbar", () => {
    const { getByTestId } = render(<HomeView />);
    expect(getByTestId("navbar")).toMatchSnapshot();
  });
});
