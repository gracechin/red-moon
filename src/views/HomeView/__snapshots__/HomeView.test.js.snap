// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView /> should render navbar 1`] = `
<div
  data-testid="navbar"
>
  <h1>
    Red Moon
  </h1>
</div>
`;

exports[`<HomeView /> should render view 1`] = `
<div
  data-testid="home-view"
>
  <div
    data-testid="navbar"
  >
    <h1>
      Red Moon
    </h1>
  </div>
</div>
`;
